duckduckgo/BrowserServicesKit
# Update autofill to fixed release/identities

Task/Issue URL: https://app.asana.com/0/392891325557410/1201884903028199
Tech Design URL:
CC:

**Description**:

Fixes broken release/identities update

**Steps to test this PR**:
1. Open https://stripe.dev/elements-examples/
2. Ensure autofill is displayed when clicking into the name field



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
